### PR TITLE
use operators grammar in yul expressions

### PR DESCRIPTION
**Pull request by OmarTawfik**

### CODE REVIEW

File: crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml
Comment (AntonyBlakey) [inline crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml:1]: nit: I don't think a declaration is a statement.

**Review by AntonyBlakey — APPROVED** [review]

File: crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml
Comment (OmarTawfik) [reply crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml:1]: Can you please clarify? Why would it not be?
It is not an expression, and has to exist next to other top level statements like assignment statements, loops, etc...

```
let x, y, z := value
```

**Comment by OmarTawfik** [comment]
Closed by accident. Created #385 instead.

File: crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml
Comment (AntonyBlakey) [reply crates/solidity/inputs/schema/grammar/06-yul/02-yul-statements/productions.yml:1]: IME declarations and statements are separate categories. But you might be right in this case. In any case, not a blocker.

**Pull request closed without merging**